test/e2e/cluster_test.go

```diff
@@ -234,8 +235,21 @@ func TestClusterDeleteDenied(t *testing.T) {
 }
 
 func TestClusterDelete(t *testing.T) {
+	// Running this test requires knowing which context to use.
+	// For Argo CD CI, the k3s context is default, but this can
+	// vary if running locally. Use ARGOCD_E2E_CONTEXT_NAME to
+	// run this test locally using a specific context name (e.g. minikube).
+	var ctxName string
+	if os.Getenv("ARGOCD_E2E_K3S") == "true" {
+		ctxName = "default"
+	} else if os.Getenv("ARGOCD_E2E_CONTEXT_NAME") != "" {
```

Comment:
Not sure if there's any process for introducing a new env var. Also very open to naming of it.